python/.changelog.d/4282.added

```diff
@@ -0,0 +1 @@
+Added shortcut in trezorlib for "academic" seed.
```

Comment:
you should specify that this is for `load-device`, which is a debug command
something perhaps like:

> Added shortcut for loading a debug device with the "academic" SLIP39 seed.

?

Reply:
[ea6ebc7](https://github.com/trezor/trezor-firmware/pull/4289/commits/ea6ebc7ff3e3fd24c8039dd606e54fca45a0e36b)